src/bp/lang-server/api.ts

```diff
@@ -15,9 +16,9 @@ export type APIOptions = {
   authToken?: string
   limitWindow: string
   limit: number
-  service: LanguageService
+  languageService: LanguageService
+  downloadManager: DownloadManager
   readOnly: boolean
```

Comment:
languageService & downloadManager shouldn't be part of APIOptions. It's mixing instances & config in a single object

Reply:
another PR